src/main/frontend/dicts.cljc

```diff
@@ -225,7 +225,7 @@
         :settings-page/custom-date-format-warning "Re-index required! Existing journal references would be broken!"
         :settings-page/preferred-file-format "Preferred file format"
         :settings-page/preferred-workflow "Preferred workflow"
-        :settings-page/preferred-pasting-file "Preferred pasting file"
+        :settings-page/preferred-pasting-file "Prefer pasting file"
```

Comment:
add 'a' i.e "Preferred pasting a file"   
The verb "to paste" requires an object, in this case "a file," to make the sentence complete.

Reply:
We are really beyond the normal rules here (there may be no definitive answer), but the article should be "the" rather than "a", however this is not a complete sentence (there is no antecedent to which "the" could refer). "Prefer pasting file" is an abbreviated phrasing, sort of like you might write on a to-do list: "buy pineapple", and it resolves the awkwardness of "the" with no antecedent.

If you have a strong opinion, you can change it. Also, thank you, and you're welcome!

Reply:
@lefth thanks for the clarification!